rich/console.py

```diff
@@ -656,8 +656,12 @@ def __init__(
 
         self.is_jupyter = _is_jupyter() if force_jupyter is None else force_jupyter
         if self.is_jupyter:
-            width = width or 93
-            height = height or 100
+            jupyter_columns = self._environ.get("JUPYTER_COLUMNS", "")
+            width = width or (
+                int(jupyter_columns) if jupyter_columns.isdigit() else 115
```

Comment:
let's move these numbers to the module level as constants. Suggest `JUPYTER_DEFAULT_COLUMNS=115`

BTW Have you tested this on lab and notebook, and could you send screenshots so I can see it fits comfortably...